cpp/include/cudf/detail/utilities/hash_functions.cuh

```diff
@@ -22,6 +22,7 @@
 
 #include "cuda_runtime_api.h"
 #include "cudf/types.hpp"
+#include "sm_35_intrinsics.h"
```

Comment:
This include doesn't look right. You should not need any additional includes to use intrinsics like funnelshift.

Reply:
Auto add by the IDE, will remove.